include/at_cmd_parser/at_cmd_data_models.h

```diff
@@ -0,0 +1,101 @@
+/**@file at_cmd_data_models.h
+ *
+ * @defgroup at_cmd_parser_data_models AT Command decoder data models
+ * @ingroup  at_cmd_parser
+ *
+ * @{
+ *
+ * @brief Data model of objects given by the Modem.
+ */
+
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+#ifndef AT_CMD_DATA_MODELS_H__
+#define AT_CMD_DATA_MODELS_H__
+
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+enum at_cmd_models {
```

Comment:
Could you please provide documentation to enums and data structures?

Reply:
Please provide doxygen comments in this file @evenl

Reply:
Yes, working on a completeish doxygen update.